status/status.go

```diff
@@ -216,3 +220,17 @@ func GenerateSummary(state intgteststat.IntegrationTestStatus, snapshotName, sce
 
 	return summary, nil
 }
+
+// GenerateURLrefferenceToConsole returns string link to integration test pipelinerun logs for tenant
```

Comment:
```suggestion
// GenerateURLreferenceToConsole returns string link to integration test pipelinerun for tenant
```
nit: there was a typo, also whatever you like `GenerateURLreferenceToConsole` or `GenerateURLReferenceToConsole`